roles/setup_assisted_installer/defaults/main.yml

```diff
@@ -41,4 +41,8 @@ image_service_host: "{{ host | default('ansible_fqdn') }}" # bastion -> image se
 use_local_mirror_registry: "{{ setup_registry_service | default(False) }}"
 
 config_file_path: /tmp/wip/config
-port: 8090
\ No newline at end of file
+port: 8090
+
+#http_proxy: ""
```

Comment:
could you remove these :)

Reply:
Yes, I dont disagree...On the other hand added them there in case someone wants to add the proxy settings, to at least know they can set them for the AI...so they have to start grepping or finding proxy related stuff. Not sure if I am explaining myself..

Reply:
Ah that's what you mean with that! 
At minimum I would add a comment to explain that. 
I think a clearer way is to use a "magic" value like an empty string and instead of checking if it is defined check it not the magic value.

Reply:
I am ok with that. Just make the changes...PTAL.